src/components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { type } from "os";
+import React from "react";
+import {  DataCardContainer,  DataCardHeader,  BottomLineStyle,  DataContent,  dataContentStyle,  chartImgStyle} from "./DataCard-style";
+import chartImg from "../../assets/Icons/chart.svg";
+import LineGraph  from './LineGraph'
+import PieGraph from "./PieGraph";
+
+type IDataCard = {
+  title: string;
+  type: string;
+};
+
+const DataCard = (props: IDataCard) => {
+  const { title, type } = props;
+
+  const renderGraph = () => {
```

Comment:
You need here an indicative name.. that function doesn't render a graph, it returns a placeholder for empty data.